### DISCUSSION TITLE
Need help with 1980s tv broadcast and vhs tape effect

### DISCUSSION THREAD

**Discussion by angryfrenchtoast** [ntsc-rs/ntsc-rs#144] [Q&A] [answered]
Can someone please tell me how to make a realistic 1980s tv broadcast effect and a 1980s vhs tape effect?
This is what i want the tv effect to look like:

![WhatsApp Image 2024-08-01 at 15 21 20](https://github.com/user-attachments/assets/0be457d7-5352-48f7-a3a8-a8afc3bb801e)

This is what i want the vhs tape effect to look like:

![image](https://github.com/user-attachments/assets/df1aeb71-54ee-4f30-a939-0c2982774343)

**Comment by Degamisu** [comment, marked as answer]
I went over this in a guide I created not to long ago! Hopefully it helps.

Also, that bottom tape appears to be Open Reel, and not VHS.

[guide](https://github.com/valadaptive/ntsc-rs/discussions/140)

**Reply by angryfrenchtoast** [reply]
Thanks! sorry for the late response.

**Reply by Degamisu** [reply]
No worries! I'm glad I could help. Good luck with the preset!
